dispatch/apps/api/serializers.py

```diff
@@ -38,6 +28,54 @@ class Meta:
             'image'
         )
 
+class UserSerializer(DispatchModelSerializer):
+    """
+    Serializes the User model.
+    """
+
+    email = serializers.EmailField(
+        required=True,
+        validators=[UniqueValidator(queryset=User.objects.all())]
```

Comment:
I too thought DRF was supposed to automatically validate unique fields, but we were getting a database `IntegrityError` until we explicitly defined the validator. 😕